packages/wrangler/src/__tests__/user.test.ts

```diff
@@ -78,20 +86,19 @@ describe("User", () => {
 				oauth_token: "some-oauth-tok",
 				refresh_token: "some-refresh-tok",
 			});
-			const outcome = mockRevokeAuthorization();
-
-			await runWrangler("logout");
-
-			expect(outcome.actual.url).toEqual(
-				"https://dash.cloudflare.com/oauth2/revoke"
+			let counter = 0;
+			msw.use(
+				rest.post("*/oauth2/token/revoke", (_, response, context) => {
+					counter++;
+					// Make sure that we made the request to logout.
+					expect(counter).toBe(1);
```

Comment:
Move to after lin 99?

Reply:
Why? The MSW handler is invoked after that the Wrangler command is executed.

Reply:
What if Wrangler was broken and the request to `token/revoke` is never made?
This test would still pass, no?

Reply:
🤔